conf/susi/en_0090_fail.json

```diff
@@ -22,12 +22,12 @@
   }, {
    "comment":"this is the catch-all rule which fires if no other rule applies",
    "phrases":[ {"type":"pattern", "expression":"*"} ],
-   "process":[ {"type":"console", "expression":"SELECT title,description,link FROM yacy WHERE query='$1$';"}],
+   "process":[ {"type":"console", "expression":"SELECT title,description,link,image FROM yacy WHERE query='$1$';"}],
    "actions":[
      {"type":"answer", "select":"random",
       "phrases":["I found this on the web:"],
       "mood": "sabta"},
-     {"type":"rss", "title": "title", "description": "description", "link": "link", "count": 3}
+     {"type":"rss", "title": "title", "description": "description", "link": "link", "image": "image", "count": 3}
```

Comment:
@DravitLochan Here the count attribute is not meant for the number of keys in the object. It is used to tell the client as to how many results must be rendered from answers[0].data